packages/graphiql-react/vite.config.ts

```diff
@@ -11,12 +12,18 @@ export default defineConfig({
       titleProp: true,
     }),
   ],
+  css: {
+    postcss: {
+      plugins: [postCssNestingPlugin()],
+    },
+  },
   esbuild: {
     // We use function names for generating readable error messages, so we want
     // them to be preserved when building and minifying.
     keepNames: true,
   },
   build: {
+    sourcemap: true,
```

Comment:
should we still include `.ts` files in the bundle?https://discord.com/channels/625400653321076807/966768858402816020/1080508965768667236